src/views/virtualmachines/list/VirtualMachinesList.scss

```diff
@@ -3,3 +3,15 @@
     height: fit-content;
   }
 }
+
+.vm-listpagebody .pf-v5-c-table {
+  .pf-v5-c-table__th.pf-v5-c-table__check {
+    width: 50px !important;
```

Comment:
we need the important? can add a more nested selector?

Reply:
Result without important

<img width="1625" alt="Screenshot 2024-11-08 at 10 35 43" src="https://github.com/user-attachments/assets/3a7a4187-7704-404c-9a14-53ac992accdb">